qpylib/qpylib.py

```diff
@@ -11,17 +11,18 @@
 
 # ==== Logging ====
 
-def create_log():
+def create_log(syslog_enabled=True):
     ''' Initialises logging.
         Threshold log level is set to the value of the "log_level" field
         in the app manifest.json, or INFO if that field is absent.
         Creates a file log handler which directs logs to store/log/app.log.
-        Creates a Syslog handler, but only if environment variables
-        QRADAR_CONSOLE_IP and QRADAR_APP_UUID are both set.
+        Creates a Syslog handler if syslog_enabled is True and only if
```

Comment:
This more precise: `Creates a Syslog handler, but only if syslog_enabled is True and...`